scraper/scraper/spiders/h1Spider.py

```diff
@@ -13,4 +16,5 @@ def start_requests(self):
     def parse(self, response):
         filename = "h1 headers"
         with open(filename, 'wb') as f:
-            f.write(response.xpath("//h1/text()")[0].extract())
\ No newline at end of file
+            f.write(response.xpath("//h1/text()")[0].extract())
+            
```

Comment:
@daniellevi1 You have spaces here. Notice the red mark in GitHub, it should be a blank line, with no spaces at all.